data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
The image had a strange warping effect, as seen in the screenshot in #15. I think that the culprit of that multiple rules competing with each other, but I'm not sure.